docs/source/release_notes/release-notes.rst

```diff
@@ -45,6 +45,12 @@ v0.22.0
          - The ``clean-cass-locks-state`` CLI now clears the lock instead of dropping the locks table.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1056>`__)
 
+    *    - |new|
+         - Added the ability to do custom performance logging for Sweep. This is currently only available for
```

Comment:
Internal products that use our metrics library could presumably want to integrate with that.  The issue is the metrics library is not public.  If we changed slf4j logging to be in a structured format (like json) instead of text we might be able to get away without making a custom logger interface and rely on using a custom logger which parses the json.  The custom logger interface makes it possible to avoid needing to serialize the log messages and then deserialize them just to integrate with our metrics library.